lib/ezutils/classes/ezextension.php

```diff
@@ -547,8 +547,13 @@ public static function getHandlerClass( ezpExtensionOptions $options )
                 // only use reflection if we have params to avoid exception on objects withouth constructor
```

Comment:
this should probably be removed, as it's ... wrong:
using `newInstanceArgs()` with parameters on classes without a constructor will result on a fatal error.